packages/multichain-transactions-controller/CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+- Bump `@metamask/keyring-internal-api` from `^5.0.0` to `^6.0.0` ([#5347](https://github.com/MetaMask/core/pull/5347))
```

Comment:
I missed `BREAKING` on some of these but will fix all of them in the release PR, to avoid additional reviews.